Formula/k/k3d.rb

```diff
@@ -2,8 +2,8 @@ class K3d < Formula
   desc "Little helper to run CNCF's k3s in Docker"
   homepage "https://k3d.io"
   url "https://github.com/k3d-io/k3d.git",
-    tag:      "v5.6.0",
-    revision: "9748b1e158f3a03e807c6a989edc0fee856ff5a2"
+    tag:      "v5.6.2",
+    revision: "6cda9e0993c8759e857a1b9bdfde28b7c61e61dc"
```

Comment:
![image](https://github.com/Homebrew/homebrew-core/assets/1580956/6f28e412-7d21-4b06-80ad-f942789d2276)
```suggestion
    revision: "b2c790b51fa038e9804395fcf914df7d00128605"
```